airqo-platform/AirQo-api
# [chore] Containerisation of data manager microservice

- What does this PR do?
This PR introduces the docker files used for containerisation of the data management service.

- What JIRA task is related to this PR?
No link available

- How do I test out this PR?
Git clone this repo `cd AirQo-api/src/data-mgt`
Create a docker image locally: `docker build -t us.gcr.io/airqo-250220/airqo-data-mgt-api .`
Afterwards, run the newly created image: `docker run -d -p 5000:3000 us.gcr.io/airqo-250220/airqo-data-mgt-api`
Using your REST client, test out all the endpoints for the` data-mgt`, use port 5000 as specified above or change to what you would like to use.
This PR requires Redis to be running in the background in order to work.

- Which end points should are ready for testing?
All the endpoints in the `data-mgt` sheet of the [API documentation](https://docs.google.com/spreadsheets/d/1-NGTDBtTcIi8Fg2UYSWjcK6geKrZdopz5kxRFsCEMPs/edit#gid=2081265667).

- **OPTIONAL for testing:** 
One could also push to the Google Container Registry using `docker push us.gcr.io/airqo-250220/airqo-data-mgt-api`
